### PR TITLE
updateFooter and add main search section

### PR DESCRIPTION
**Pull request by mareann**
### Describe The Problem Being Solved:
Skeleton of home page recipe app
smaller page
rlc_homepage_small
regular size page
rlc_homepage

### Checklist For Submitter

* [ x] I have documented all new functionality
* [x ] I have screenshot new UI changes and attached them to this PR

### Checklist For Reviewer
* [ ] Code formatting/static analysis: is the code formatted properly? Does a linter/other form of static analysis reveal any issues? Does the code have appropriately low cyclomatic complexity?
* [ ] Code architecture: concerns are separated, code follows single-responsibility principle, code utilizes OOP, DRY code.
* [ ] Coding best practices: code avoids hard-coded variables, inefficient computations, reinventing the wheel when using frameworks. Code is well-commented and generally readable.
* [ ] Non-functional requirements: Code has accompanying tests (if using TDD). Programmer can easily explain decisions to reviewers.

### CODE REVIEW

File: frontend/src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    text-align: center;
+    padding-top: 12px;
+    font-weight: 500;
+}
+
+.designByCo {
+    font-size: 1vw;
+    color: #1f1e1e;
+    font-weight: 600;
+   /* padding-bottom: 5vw;*/
```
Comment (zachgarwood) [inline frontend/src/components/Footer/Footer.css]: I think commented out code is unnecessary clutter, let's get rid of it. We can always add it back in if we decide we need it.

File: frontend/src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    font-weight: 400;
+    opacity: 0.7;
+    color: #666565;
+}
+
+.containerSection {
+   /* width: 100%;
+    height: "50px",
+    margin-left: 0;
+    margin-right: 0;*/
```
Comment (zachgarwood) [inline frontend/src/components/Footer/Footer.css]: More commented out code.

File: frontend/src/components/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    padding-top: 1vw;
+    font-weight: 600;
+    padding-left: 0;
+    padding-right: 0;
+    /*padding-top: "20vw";*/
```
Comment (zachgarwood) [inline frontend/src/components/Footer/Footer.css]: And more.

File: frontend/src/components/MainSearch/MainSearch.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
+.mainSearchContainer {
+   /* height: 40vw; */
```
Comment (zachgarwood) [inline frontend/src/components/MainSearch/MainSearch.css]: And here

File: frontend/src/components/MainSearch/MainSearch.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    background-repeat: no-repeat;
+    padding-bottom: 0;
+}
+
+.search {
+   
+    width: 100vw;
+    font-size: 2vw;
+ /*    color: rgba(240, 235, 235, 0.2);
+    font-weight: 100; */
```
Comment (zachgarwood) [inline frontend/src/components/MainSearch/MainSearch.css]: And here

File: frontend/src/components/Footer/Footer.css (hunk 1 above)
Comment (mareann) [reply frontend/src/components/Footer/Footer.css]: Yes. Thanks. Forgot to remove.

File: frontend/src/components/MainSearch/MainSearch.css (hunk 4 above)
Comment (mareann) [reply frontend/src/components/MainSearch/MainSearch.css]: removed. thanks

File: frontend/src/components/MainSearch/MainSearch.css (hunk 5 above)
Comment (mareann) [reply frontend/src/components/MainSearch/MainSearch.css]: done

File: frontend/src/components/Footer/Footer.css (hunk 2 above)
Comment (mareann) [reply frontend/src/components/Footer/Footer.css]: done

**Pull request merged**